src/plasma/test/test.py

```diff
@@ -250,18 +250,39 @@ def test_evict(self):
     self.assertEqual(client.evict(2000), 996 + 995 + 994)
 
   def test_subscribe(self):
+    import uuid
     # Subscribe to notifications from the Plasma Store.
     sock = self.plasma_client.subscribe()
     for i in [1, 10, 100, 1000, 10000, 100000]:
       object_ids = [random_object_id() for _ in range(i)]
+      metadata_list = []
+      datasize_list = []
       for object_id in object_ids:
         # Create an object and seal it to trigger a notification.
-        self.plasma_client.create(object_id, 1000)
+        object_str = str(uuid.uuid4())
+        object_str_len = random.randint(0,len(object_str))
+        object_str = object_str[:object_str_len]
```

Comment:
metadata_size_list = [np.random.randint(1000) for _ in range(i)]
data_size_list = [np.random.randint(1000) for _ in range(i)]

for j in range(i):

 (object_ids[i], size=data_size_list[i], metadata=bytearray(np.random.bytes(metadata_size_list[i])))